gsy_framework/schemas.py

```diff
@@ -79,7 +80,32 @@ class ScenarioSchemas:
                             {"type": "null"},
                             {"type": "array"},
                             {"type": "string"}]},
-                    "use_market_maker_rate": {"type": "boolean"}
+                    "use_market_maker_rate": {"type": "boolean"},
+                    "powerProfileUUID": {"type": "string"}
+                }
+            },
+            "wind_turbine": {
+                "type": "object",
+                "properties": {
+                    "name": {"type": "string"},
+                    "type": {"enum": ["WindTurbine"]},
+                    "uuid": {"type": "string"},
+                    "libraryUUID": {"anyOf": [{"type": "string"}, {"type": "null"}]},
+                    "initial_selling_rate": {"anyOf": [{"type": "number"}, {"type": "null"}]},
+                    "final_selling_rate": {"type": "number"},
+                    "fit_to_limit": {"type": "boolean"},
+                    "update_interval": {"anyOf": [{"type": "number"}, {"type": "null"}]},
+                    "energy_rate_decrease_per_update": {"anyOf": [{"type": "number"},
+                                                                  {"type": "null"}]},
+                    "power_profile": {
+                        "anyOf": [
+                            {"type": "object"},
+                            {"type": "number"},
+                            {"type": "null"},
+                            {"type": "array"},
+                            {"type": "string"}]},
+                    "use_market_maker_rate": {"type": "boolean"},
+                    "powerProfileUUID": {"type": "string"}
```

Comment:
Should the capacity_kW not be included here too, in order to pass through the schema validator ?

Reply:
It should indeed. Shows that I did not test it end to end again 🙈 . Let me add it